FWCore/MessageLogger/interface/ErrorObj.h

```diff
@@ -45,7 +45,7 @@ namespace edm {
   public:
     // --- birth/death:
     //
-    ErrorObj(const ELseverityLevel& sev, const ELstring& id, bool verbatim = false);
+    ErrorObj(const ELseverityLevel& sev, std::string_view id, bool verbatim = false);
```

Comment:
@Dr15Jones Given that `ELseverityLevel` is essentially an `int`, I wonder if we should pass it around by value instead of by reference?

Reply:
Passing by value makes sense to me. (Much of the internals of MessageLogger are C++98 era of code)